src/lib/loaders/loaders_with_authentication/gravity.ts

```diff
@@ -72,6 +73,7 @@ export default (accessToken, userID, opts) => {
       "is_saved"
     ),
     savedArtworksLoader: gravityLoader("collection/saved-artwork/artworks", { user_id: userID, private: true }),
+    searchLoader: searchLoader(gravityLoader),
```

Comment:
Not sure that we even need this authenticated loader (search shouldn't really change based on your access/identification IMO, i wouldnt think).

Reply:
Yep, looks like there isn't any difference :+1: